src/modules/reports/models/reports.dto.ts

```diff
@@ -0,0 +1,4 @@
+export class GetPatientAppointmentsReportsDTO {
+  startedAt: string;
```

Comment:
can u add the swagger documentation for the DTO? you missed the 

`@ApiProperty({
    description: "description of the property",
    example: 'Example',
  })`

Reply:
for both values

Reply:
I have already made the changes